msteams-platform/overview.md

```diff
@@ -1,100 +1,55 @@
+> [!TIP]
+> Don't have Microsoft Teams? Get a free Office 365 developer subscription or activate it for your existing Office 365 account. See [Set up your Office 365 tenant](~/get-started/get-started#1-set-up-your-office-365-tenant).
 
----
-
-[Get started with bots.](~/concepts/bots/bots-overview)
-
----
-
-![Example of a bot assisting a user](~/assets/images/bot_example.png)
+## What you'll find in these topics
 
-### Compose extensions
+Are you new to the Microsoft Teams developer platform? Choose one of the following walkthroughs to quickly learn the basics of developing an app for Microsoft Teams.
 
->[Public Developer Preview only](~/reference/general/developer-preview)
+*	[Get started with .NET and C#](~/get-started/get-started-dotnet)
+*	[Get started with Node.js](~/get-started/get-started-nodejs)
 
-Leverage the power of your web service to provide a quick and easy way for your users to insert your content into the chat stream.
-
----
-
-[Get started with compose extensions.](~/concepts/compose-extensions)
-
----
+The rest of this documentation provides the information you need to extend Microsoft Teams by adding your content and services.
 
-![Example of a compose extension](~/assets/images/composeextension/ceoverviewexample.png)
+* **Concepts**&emsp;Learn more about creating Teams apps, and find everything you need to know about the entire range of capabilities in Teams: tabs, bots, Connectors, compose extensions, and more.
+* **Publishing**&emsp;After you develop your app, you need to test and publish it. These topics explain how.
+* **Scenarios**&emsp;Go deep into the specifics of conversations with bots.
+* **Reference**&emsp;Learn how to design a great Teams app, get the details about our SDKs, and more.
+* **Troubleshooting**&emsp;If your app isn't behaving as expected, check here.
+* **FAQ**&emsp;Everyone has them; here are our frequently asked questions.
+* **Samples**&emsp;See how others created Teams apps.
+* **Feedback**&emsp;Didn't find what you need? Let us know!
 
-### Connectors
-
->[Public Developer Preview only](~/reference/general/developer-preview)
-
-Office 365 Connectors are a great way to get useful information and content into Microsoft Teams.
-
-You can integrate your service by posting rich Connector cards into custom incoming webhooks.
-
----
-
-[Get started with Connectors.](~/concepts/connectors)
+## Apps in Microsoft Teams
 
----
+To extend Microsoft Teams by adding your content and services, you can create a Microsoft Teams app&mdash;a package of capabilities that you host and that can be distributed through Microsoft Teams or installed by individual teams.
 
-![Gallery of Connectors](~/assets/images/connector_example.png)
-<!-- TODO - update image to latest -->
+To learn about Teams apps and all the possibilities they offer, see [Develop apps for Microsoft Teams](~/concepts/apps/apps-overview).
 
 ## Custom bots
```

Comment:
I don't think this should be here at all .... Too much emphasis ...

Reply:
Sold!